projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.spec.ts

```diff
@@ -104,4 +113,36 @@ describe('SearchAiResultsComponent', () => {
       expect(component.hasError).toBe(true);
     });
   });
+
+  describe('query params change', () => {
+    beforeEach(() => {
+      searchAiService = TestBed.inject(SearchAiService);
```

Comment:
You assign this service here but that variable is declared in top describe. Can you move declaration of service to this inner describe as it is initialized only here?